nanotune/tests/functional_tests/test_dottuner_functional.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2021 Jana Darulova
+#
+# This software is released under the MIT License.
+# https://opensource.org/licenses/MIT
+
+import logging
+import pytest
+from nanotune.device_tuner.dottuner import DeviceState
+from nanotune.device.device_layout import DoubleDotLayout
+from nanotune.tests.functional_tests.sim_tuner import SimDotTuner
+logger = logging.getLogger(__name__)
+
+
+def test_double_dot_tuning_good_example(
+        tuner_default_input, sim_device, sim_scenario_dottuning
+    ):
+    device = sim_device
+    self = SimDotTuner(
+        **tuner_default_input, sim_scenario=sim_scenario_dottuning,
+    )
+    device_layout = DoubleDotLayout
+    target_state = DeviceState.doubledot
+    max_iter = 10
+    take_high_res = False
+    continue_tuning = False
+
+    self.set_central_and_outer_barriers(
+        device, device_layout, target_state
+    )
+    assert device.central_barrier.voltage() == -1.42947649216405
+    assert device.left_barrier.voltage() == -0.5141713904634877
+    assert device.right_barrier.voltage() == -0.5478492830943646
```

Comment:
you may want to consider using something like numpys assert almost equal here. Floating point numbers are often hard to exactly reproduce due to round of errors and so on